plugin/builtin/patchtransformer/PatchTransformer.go

```diff
@@ -100,6 +100,11 @@ func (p *plugin) transformStrategicMerge(m resmap.ResMap, patch *resource.Resour
 	if err != nil {
 		return err
 	}
+
+	if p.Options["allowNoTargetMatch"] {
+		log.Println("Warning: patches target not found for Target")
```

Comment:
Can we print for users the "Target" that we're looking for?

As a user with a complex Kustomization, I'd be confused and need to debug which patch target is missing.